osquery/tables/system/windows/windows_eventlog.cpp

```diff
@@ -0,0 +1,271 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <Windows.h>
+#include <winevt.h>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/xml_parser.hpp>
+
+#include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
+
+#include <osquery/core/windows/wmi.h>
+#include <osquery/events/windows/windowseventlogparser.h>
+#include <osquery/sql/dynamic_table_row.h>
+#include <osquery/tables/system/windows/windows_eventlog.h>
+#include <osquery/utils/conversions/join.h>
+#include <osquery/utils/conversions/split.h>
+#include <osquery/utils/conversions/windows/strings.h>
+
+namespace pt = boost::property_tree;
+
+namespace osquery {
+namespace tables {
+
+const std::string kEventLogXmlPrefix = "<QueryList><Query Id=\"0\">";
+const std::string kEventLogXmlSuffix = "</Query></QueryList>";
+
+const int kNumEventsBlock = 1024;
+
+Status parseWelXml(QueryContext& context, std::wstring& xml_event, Row& row) {
+  pt::ptree propTree;
+  WELEvent windows_event;
+  auto xml_status = parseWindowsEventLogXML(propTree, xml_event);
+  if (!xml_status.ok()) {
+    VLOG(1) << "parseWelXml : " << xml_status.toString();
```

Comment:
We should keep `VLOG` user friendly, something slightly between INFO and DEBUG.

I recommend `"Error parsing event log XML: " << xml_status.toString(); `